src/app/pages/test-report/components/reverse-diagram-modal/__tests__/reverse-diagram-modal.spec.ts

```diff
@@ -0,0 +1,217 @@
+/* eslint-disable */
```

Comment:
not sure we should be disabling eslint for entire pages

Reply:
yeah good point have disabled just for spec that is throwing error. don't want to re-write at this point